docs/buildah-push.md

```diff
@@ -61,6 +61,10 @@ The [username[:password]] to use to authenticate with the registry if required.
 If one or both values are not supplied, a command line prompt will appear and the
 value can be entered.  The password is entered without echo.
 
+**--digestfile** *Digestfile*
+
+Write the digest to the file.
```

Comment:
“The digest of the resulting image”, ”the digest of the image created at the destination”, or something like that? “The digest” does not really say which one (and the value may differ from the digest of the source).

Not sure, maybe something like “After copying the image” could be useful to indicate that this does not prevent the default action (pushing an image) from happening. OTOH the example added below is clear enough about that.

Reply:
Changed to:
`After copying the image, write the digest of the resulting image to the file.`